src/components/Camera/style.css

```diff
@@ -103,6 +103,18 @@
   /*Mirroring transform: scale(-1, 1); */
 }
 
+/* this will only apply styles to Edge */
+@supports (object-fit: cover) and (-ms-ime-align:auto) {
+  .video {
+    height:100%;
```

Comment:
Minor: redundant, since same property is applied before

Reply:
Yes